### PR TITLE
chore(ci): Update Github workflows from templates

### PR DESCRIPTION
**Pull request by mejo-**

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+            src:
+              - '.github/workflows/**'
+              - 'src/**'
+              - 'appinfo/info.xml'
+              - 'package.json'
+              - 'package-lock.json'
+              - 'tsconfig.json'
+              - '**.js'
+              - '**.ts'
+              - '**.vue'
```
Comment (max-nextcloud) [inline .github/workflows/cypress.yml:38]: I think we should also run cypress on changes to the backen - i.e. `lib`.

Comment (max-nextcloud) [reply .github/workflows/cypress.yml:38]: While you're at it you could make the workflow expression more precise and only run on changes to the cypress workflow.

**Review by max-nextcloud — APPROVED** [review]
Two minor comments. Other than that looks good.

Thanks for taking care of this :heart:

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (mejo-) [reply .github/workflows/cypress.yml:38]: Well, the workflow regex is the default from all upstream templates, I'd rather keep it that way to not diverge more. But agreed, `lib` should be added. Good catch :ok_hand:

**Pull request merged**